### ISSUE TITLE
Keyboard only mode

### ISSUE THREAD

**Issue by jan-kleks** [KyleU/solitaire.gg#37]
While experimenting with game pad support, you can also think how to make it possible to use just the keyboard to play your solitaires. If there is a [netbook](https://en.wikipedia.org/wiki/Netbook) with a small touch-pad (I generally don't like touch-pads) and no mouse around it's a pain to play solitaire on such a device.

Things to consider:
1) Using either arrow keys, WSAD, or numpad for navigation
2) A way to select/deselect cards, and some sort of visual feedback for card selection; a border around the card, an arrow or highlighting
3) Other shortcuts that would facilitate moving around the tableau like key combinations to quickly move to the first and the last card column or adding letters or numbers to the tableau (in keyboard mode only, of course) which when pressed on the keyboard would move you to a given position/column.

**Comment by KyleU** [comment]
Yep, the gamepad code and keyboard code share logic. Once I figure out the gamepad logic, we get keyboard bindings for free.
